backend/services/implementations/userService.ts

```diff
@@ -113,9 +121,18 @@ class UserService implements IUserService {
       email: firebaseUser.email ?? "",
       role: user.role,
       town: user.town,
+      emailVerified: firebaseUser.emailVerified,
     };
   }
 
+  async getUserByEmail(email: string): Promise<UserDTO> {
+    const {
+      emailVerified,
+      ...rest
+    } = await this.getUserWithVerificationStatusByEmail(email);
+    return rest;
+  }
+
```

Comment:
Why not just combine it into a single endpoint for user + user email verified? Is there a situation where we want one but not the other?

Reply:
Hmmm, I think I'm approaching this from a REST perspective, but I find this user is a separate entity as the users we use elsewhere in general. But honestly, this probably fits the graphql paradigm more and we can just not specify emailVerified in the query.

Reply:
I'll add it to the UserDTO since the frontend deals with the data differently than the backend anyway.